### ISSUE TITLE
The future of this project

### ISSUE THREAD

**Issue by electron271** [popcorn-kernel/popcorn#80]
Hello everybody, due to 3 developers and contributors leaving the team I think I should make an announcement detailing everything that happened and the future of this project. So recently 2 developers stepped down and 1 contributor was removed from the project, as such we are in need of new developers. Many developers (including myself) have been busy with a lot of life stuff and unrelated projects as such lately development has been stalled, so I am calling for new developers for this project. I will try my best to develop this project again when I have time, however I am unsure how much I myself can do. It is very possible this project will have to go on hold for the time being.

_tldr; developers/contributors have left and we need new ones, project may be going on hold_

Sorry for not the best write up on this all as it is 1 AM.

**Comment by janTatesa** [comment]
I am leaving too, goodbye.

**Comment by sparkles-laurel** [comment]
im stepping down too
i'm stepping down because i have to study for [YKS](https://en.wikipedia.org/wiki/Student_Selection_and_Placement_System)

**Comment by SatoCodingStuff** [comment]
kernel going into another way.

**Comment by electron271** [comment]
i will keep this issue open for now

**Comment by electron271** [comment]
until things are more resolved

**Comment by SatoCodingStuff** [comment]
I will be resigning from the project.

**Comment by Lily8000** [comment]
I have told you me and a other friend ate willing to help the project.
